app/models/game.rb

```diff
@@ -1,5 +1,7 @@
 class Game < ActiveRecord::Base
   after_create :populate_board
+  after_create :first_move
+  after_create :assign_pieces, :if => :is_full?
```

Comment:
this isn't actually working, your test works because you create the game with both white and black user_id's assigned. you'll probably want to call the game.assign_pieces from the game update action when the black user is assigned.